docs/api.md

```diff
@@ -99,100 +98,104 @@ nos.claimGas()
 const nos = window.NOS.V1;
 
 const scriptHash = '2f228c37687d474d0a65d7d82d4ebf8a24a3fcbc';
 const operation = '9937f74e-1edc-40ae-96ad-1120166eab1b';
-const arg1 = 'ef68bcda-2892-491a-a7e6-9c4cb1a11732';
-// const arg2 = '...' if you want to add more args, just append it to the function
+const args = ['ef68bcda-2892-491a-a7e6-9c4cb1a11732'];
 
-nos.invoke(scriptHash, operation, arg1)
+// If you handle encoding yourself, use:
+// nos.invoke({ scriptHash, operation, args, encodeArgs: false })
+nos.invoke({ scriptHash, operation, args })
     .then((txid) => alert(`Invoke txid: ${txid} `))
     .catch((err) => alert(`Error: ${err.message}`));
 ```
 
-## `getStorage`
+### `getStorage`
 The `getStorage` function retrieves the value for a specified key from a specified smart contract.
 It does not require the user to grant permission.
 
-### Parameters
-1. `string` - The script hash of a deployed Smart Contract.
-1. `string` - The key to retrieve from the Smart Contract.
-1. `object` - An optional options object
+#### Parameters
+1. `scriptHash` [string] - The script hash of a deployed Smart Contract.
+2. `key` [string] - The key to retrieve from the Smart Contract.
+3. `encodeInput` [boolean] (Optional) - A flag detailing whether or not to encode the input. This is `true` by default
+4. `decodeOutput` [boolean] (Optional) - A flag detailing whether or not to decode the output. This is `true` by default
 
-### Options
-1. **encode**: `bool` - Whether or not the input should be encoded (string to reverse hex). Defaults to true.
+#### Returns
+[any] - The stored value or `null` if the key did not contain a value.
 
-### Returns
-`any` - The stored value or `null` if the key did not contain a value.
-
-### Example
+#### Example
 ```javascript
 const nos = window.NOS.V1;
 
 const scriptHash = '85e9cc1f18fcebf9eb8211a128807e38d094542a';
 const key = 'post.latest';
-const options = { options: false }
 
-nos.getStorage(scriptHash, key, options)
+// If you want to handle encoding / decoding yourself, use:
+// nos.getStorage({ scriptHash, key, encodeInput: false, decodeOutput: false })
+nos.getStorage({ scriptHash, key })
     .then((data) => alert(`Get storage data: ${data} `))
     .catch((err) => alert(`Error: ${err.message}`));
 ```
 
-## `send`
+### `send`
 The `send` function creates a contract transaction to send assets (NEO or GAS) to a specified
 address on behalf of the currently authenticated account.  It requires the user to grant permission.
 
-### Parameters
-1. `string` - The asset ID script hash.
-1. `string` - The amount of the asset to send.  It is recommended that strings are used instead of
-   numbers to prevent floating point rounding issues.
-1. `string` - The recipient address of the asset.
+#### Parameters
+1. `asset` [string] - The asset ID script hash.
+2. `amount` [string] - The amount of the asset to send.
+**NOTE: It is recommended that strings are used instead of numbers to prevent floating point rounding issues.**
```

Comment:
I think we need to either append this to the previous line or indent it several spaces to prevent the ordered list from being interrupted.